PFR/ForMathlib/Entropy/RuzsaDist.lean

```diff
@@ -795,16 +806,19 @@ lemma condRuzsaDist_of_copy {X : Ω → G} (hX : Measurable X) {Z : Ω → S} (h
     all_goals {
       rw [Measure.map_apply ‹_›]
       convert measure_empty
-      simp [← FiniteRange.range]
+      simp only [Finset.coe_union, ← FiniteRange.range, Set.compl_union, Set.preimage_inter,
+        Set.preimage_compl, Set.preimage_range, Set.compl_univ, Set.empty_inter]
       measurability
     }
-  have hfull' : Measure.prod (μ''.map Z') (μ'''.map W') ((A ×ˢ B : Finset (S × T)): Set (S × T))ᶜ = 0 := by
+  have hfull' : Measure.prod (μ''.map Z') (μ'''.map W')
+    ((A ×ˢ B : Finset (S × T)): Set (S × T))ᶜ = 0 := by
     simp only [A, B]
     apply Measure.prod_of_full_measure_finset
     all_goals {
       rw [Measure.map_apply ‹_›]
       convert measure_empty
-      simp [← FiniteRange.range]
+      simp only [Finset.coe_union, ← FiniteRange.range, Set.compl_union, Set.preimage_inter,
+        Set.preimage_compl, Set.preimage_range, Set.compl_univ, Set.inter_empty]
       measurability
```

Comment:
I consider `measurability` to be flexible input

Reply:
Done. Thanks.